### DISCUSSION TITLE
Add default maintenance tasks when adding new inventory items

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#65] [Ideas]
When adding a new inventory item to Samedis we now have to manually enter the first maintenance task(s). In my opinion it would be good to add a task of each maintenance type by default (and maybe just to be confirmed manually). This should be possible, as you know the date of adding to the inventory and the maintenance schedules for the various tasks as set in the device type. So the first maintenance task of each task type could be added. It would save a lot of manually clicks, especially when adding a larger number of new devices.

**Comment by YvesR** [comment]
I do not follow 100%. When you add a new inventory, set service intervals (or get from catalog), on the same step you can add events (maintenance) for those where you choose service-type, auto set date from intervall (now + n months) and on saving the inventory the events are created automatically.
